test/evaluate.js

```diff
@@ -183,16 +182,7 @@ describe('evaluate', async function () {
       prepareProviderRetrievalResultStats: async () => {}
     })
 
-    let point = telemetry.find(p => p.name === 'retrieval_stats_honest')
```

Comment:
tooling probably consumes `retrieval_stats_honest` - what if we mirror all properties from `retrieval_stats_all` to `retrieval_stats_honest`, we we know now that all stats are honest?

Reply:
Good point, I've reintroduced `retrieval_stats_honest` in https://github.com/CheckerNetwork/spark-evaluate/pull/572/commits/2d2e29167c4556e37ceb3251895006d739f8a67b for backwards compatibility of existing tools.